testsuites/kubernetes_versions.robot

```diff
@@ -27,44 +27,32 @@ Documentation     Verify Helm functionality on multiple Kubernetes versions.
 ...
 Library           String
 Library           OperatingSystem
-Library           ../lib/Kind.py
 Library           ../lib/Kubectl.py
 Library           ../lib/Helm.py
 Library           ../lib/Sh.py
-Suite Setup       Suite Setup
+Library           ../lib/ClusterProvider.py
 Suite Teardown    Suite Teardown
 
 *** Test Cases ***
-#Helm works with Kubernetes 1.16.1
-#    Test Helm on Kubernetes version   1.16.1
 
-Helm works with Kubernetes 1.15.3
-    Test Helm on Kubernetes version   1.15.3
+Helm Works on Kubernetes
+    Test Helm on Kubernetes version
```

Comment:
robot keyword is `Test Helm on Kubernetes version` which does not accept any argument, so if change it to `Test Helm on Kubernetes version   %{CLUSTER_VERSION}` tests would fail because of invalid robot keyword.

earlier `Test Helm on Kubernetes version   1.15.3` here `1.15.3` is the argument being passed to `Test Helm on Kubernetes version` keyword.

Reply:
well, we can revert to the previous situation when `Test Helm on Kubernetes version` was accepting argument?

Reply:
Done